README.md

```diff
@@ -39,8 +39,8 @@ Here you can find links on how to get started with Eclipse Che:
 ### Using Eclipse Che
 Here you can find references to useful documentation and hands-on guides to learn how to get the most of Eclipse Che:
 - [Customize Che workspaces for your projects](https://www.eclipse.org/che/docs/che-7/configuring-a-workspace-using-a-devfile/)
-- [Run VSCode Extensions in Che workspaces](https://www.eclipse.org/che/docs/che-7/using-a-visual-studio-code-extension-in-che)
-- [Configure Che for your teams](https://www.eclipse.org/che/docs/che-7/building-and-running-a-custom-registry-image/)
+- [Run VSCode Extensions in Che workspaces](https://www.eclipse.org/che/docs/che-7/end-user-guide/adding-a-vs-code-extension-to-a-workspace/)
+- [Configure Che for your teams](https://www.eclipse.org/che/docs/che-7/end-user-guide/making-a-workspace-portable-using-a-devfile/)
```

Comment:
You've not only fixed the link, but changed it to completely different page. I would agree that building a devfile is more important and well suited under "Using Eclipse Che", than the old link. However, I would rename the name of the link here.

Reply:
maybe even this one would be better? https://www.eclipse.org/che/docs/che-7/end-user-guide/creating-and-configuring-a-new-workspace/ or both? WDYT?

Reply:
Both links sounds good. Here's a replacement PR: https://github.com/eclipse/che/pull/18835